src/app-layout/__integ__/app-layout-split-panel.test.ts

```diff
@@ -224,11 +224,13 @@ describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)(
       // Simulating 200% zoom on medium screens (1366x768 / 2 ~= 680x360 ).
       await page.setWindowSize({ width: 680, height: 360 });
       await page.openPanel();
-      const { height } = await page.getViewportSize();
-      await page.dragResizerTo({ x: 0, y: height });
+      const { height: screenHeight } = await page.getViewportSize();
+      const headerRect = await page.getBoundingBox('#h');
+      await page.dragResizerTo({ x: 0, y: screenHeight });
       expect((await page.getSplitPanelSize()).height).toEqual(160);
       await page.dragResizerTo({ x: 0, y: 0 });
-      expect(Math.round((await page.getSplitPanelSize()).height)).toEqual(277);
+      const splitPanelMaxSize = screenHeight - 40 - headerRect.height;
```

Comment:
Why 40?

Reply:
It is the value of CONSTRAINED_MAIN_PANEL_MIN_HEIGHT, used here: https://github.com/cloudscape-design/components/blob/main/src/app-layout/classic.tsx#L270

Reply:
Then should we use that variable directly here? or at least add a comment?

Reply:
I hesitate to do either, to be honest, as both solutions create a binding between the test and the implementation. I think the fact the assertion is now a formula is good because it makes the intent better clear and we can even define the test against multiple screen heights to ensure the formula holds. The constant 40 is just a part of that. Technically, the implementation can change, e.g. we can move it to the different place in code or compute the constant as a sum of two smaller ones - but that should not matter unless the actual value changes, so it is the actual value that is here important.